tensorflow_lite_support/python/test/task/vision/image_segmenter_test.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for image_segmenter."""
+
+import enum
+import json
+
+import numpy as np
+from absl.testing import parameterized
+from google.protobuf import json_format
+# TODO(b/220067158): Change to import tensorflow and leverage tf.test once
+# fixed the dependency issue.
+import unittest
+from tensorflow_lite_support.python.task.core.proto import base_options_pb2
+from tensorflow_lite_support.python.task.processor.proto import segmentations_pb2
+from tensorflow_lite_support.python.task.processor.proto import segmentation_options_pb2
+from tensorflow_lite_support.python.task.vision import image_segmenter
+from tensorflow_lite_support.python.task.vision.core import tensor_image
+from tensorflow_lite_support.python.test import base_test
+from tensorflow_lite_support.python.test import test_util
+
+_BaseOptions = base_options_pb2.BaseOptions
+_OutputType = segmentation_options_pb2.OutputType
+_ImageSegmenter = image_segmenter.ImageSegmenter
+_ImageSegmenterOptions = image_segmenter.ImageSegmenterOptions
+
+_MODEL_FILE = 'deeplabv3.tflite'
+_IMAGE_FILE = 'segmentation_input_rotation0.jpg'
+_SEGMENTATION_FILE = 'segmentation_golden_rotation0.png'
```

Comment:
@khanhlvg I've updated the tests to use existing data now.

Reply:
LGTM. Thanks!